### PR TITLE
Zombie Animal Upgrades

### PR DESCRIPTION
**Pull request by LyleSY**
#### Summary

SUMMARY: Content "Two upgraded monsters each for three zombie animals"

#### Purpose of change

Provide more monster variety in the wilderness

#### Describe the solution

Two new zombear upgrades, two new antlered horror upgrades, and two new pouncer upgrades. Each monster gets its own upgrade group and each upgrades into its group.

#### Describe alternatives you've considered

Wolf, dog, more

#### Testing

All spawn, no errors
image

#### Additional context

Based on a year old idea from @I-am-Erk

### CODE REVIEW

File: data/json/monsters/zombear_upgrade.json [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    "type": "MONSTER",
+    "name": { "str": "skeletal zombear" },
+    "copy-from": "mon_zombear",
+    "description": "A monstrous overgrowth of ossified tissue has replaced this zombear's rotting skin with an organic armor of dense bone.  Large clumps of black goo seep from its joints as it shambles aimlessly, with sickening crackling sounds filling the air around it.",
+    "hp": 70,
+    "speed": 100,
+    "material": [ "bone" ],
+    "color": "white",
+    "melee_dice": 5,
+    "melee_dice_sides": 9
```
Comment (LaVeyanFiend) [inline data/json/monsters/zombear_upgrade.json:84]:
```suggestion
    "melee_dice_sides": 9,
```

**Comment by Night-Pryanik** [comment]
While I welcome the idea of zombie animal upgrades, I don't like the very principle of creating a new monster by combining features from two previously existed monsters.

**Comment by I-am-Erk** [comment]
In general monster upgrades need to be done by adding templates to an existing monster instead of a new type, but nobody has added that feature yet.

I don't recommend making a zombear for every flavour of the month. I'd suggest picking two subtypes of zombie and applying them to zombears. You could make upgrades for some of the other zombie animals using other templates. Like, you could do a skeletal and an acid zombear, a grabber and a shocker cougar, a smoker and a boomer moose, etc.

As it stands this creates a precedence that will lead to far too many zombie animal variants considering how comparatively rare these special zombies are.

**Comment by LyleSY** [comment]
I am sorry to lose the Zombear Master, I really liked that image of a zombie bear rakshasa merrily roaming the woods on two legs.

**Comment by I-am-Erk** [comment]
I still think six zombears is too many. There just aren't enough bears in the game, this adds a lot of variation to what is ultimately a rare enemy. Keep the six you picked, but make 2 a cougar and 2 a moose, perhaps. Heck, maybe add a necromancer and a boomer back and then put two of those variants onto wolves, too.

As it stands, since there are many other types of zombie wildlife, doing this many beariant zombies could easily lead us down the road of having dozens to hundreds of specific subtype zombies for zombies that are seen only a few times in even a long playthrough. 

When it's possible to have simple template-based updates to zombies, we can eventually get this whole package, but I think it's inappropriate to go this far when each monster has to be individually defined.

**Comment by I-am-Erk** [comment]
All right! Zombie moose brute! Now you will know why you fear the night.

Let's go kill us some unsuspecting players eh?

**Comment by Mister-Cultist** [comment]
Great work, I've been looking foreward to more variety in the none human zombies.

**Pull request merged**